setup.go

```diff
@@ -94,6 +95,22 @@ func setup(c *caddy.Controller) error {
 			if len(args) == 1 {
 				fp.probeResistDomain = args[0]
 			}
+		case "serve_pac":
```

Comment:
This plugin also generates and keeps this PAC in memory.

+1 to "probe_resistant"